packages/react-icons/src/utils/bundleIcon.tsx

```diff
@@ -2,13 +2,14 @@ import * as React from "react";
 import { iconFilledClassName, iconRegularClassName } from "./constants";
 import { FluentIconsProps } from "./FluentIconsProps.types";
 import { makeStyles, mergeClasses } from "@griffel/react";
+import { FluentIcon } from "./createFluentIcon";
 
 const useBundledIconStyles = makeStyles({
```

Comment:
We discussed this offlline, but there's a change in the `FluentIconsProps.types.ts` file:

Before: 
```ts
export type FluentIconsProps<TBaseAttributes extends (React.SVGAttributes<SVGElement> | React.HTMLAttributes<HTMLElement>) = React.SVGAttributes<SVGElement>> = 
TBaseAttributes & {
     primaryFill?: string
     className?: string
     filled?: boolean
     title?: string
}
```
After
```ts
export type FluentIconsProps<TBaseAttributes extends (React.SVGAttributes<SVGElement> | React.HTMLAttributes<HTMLElement>) = React.SVGAttributes<SVGElement>> = 
TBaseAttributes & React.RefAttributes<SVGElement> & {
     primaryFill?: string
     className?: string
     filled?: boolean
     title?: string
}
```
The main change is the addition of `React.RefAttributes<SVGElement>` in the type